cli/app/ui_scroll_keys_test.go

```
@@ -73,21 +361,69 @@ func TestDetailModeMouseWheelScrollTranscript(t *testing.T) {
 	}
 	m = updateUIModel(t, m, tea.KeyMsg{Type: tea.KeyShiftTab})
 
-	initial := stripANSIAndTrimRight(m.view.View())
+	initial := stripDetailSelectionRail(stripANSIAndTrimRight(m.view.View()))
 	if initial == "" {
 		t.Fatal("expected detail transcript visible before mouse scrolling")
 	}
 
 	m = updateUIModel(t, m, tea.MouseMsg{Button: tea.MouseButtonWheelUp, Type: tea.MouseWheelUp})
```

golangci-lint (2.11.4)

[error] SA1019: tea.MouseWheelUp is deprecated: Use MouseAction & MouseButton instead. 

(staticcheck)